test/src/utils/index.spec.ts

```diff
@@ -186,6 +194,72 @@ describe("UTILS Index tests suites", () => {
 
     // ************************************************************ //
 
+    describe("setTeamsData(...)", () => {
+
+        const membersUrl = `${MOCK_REPOS_TEAMS_DATA[0]["url"]}/members`;
+        const repositoriesUrl = MOCK_REPOS_TEAMS_DATA[0]["repositories_url"];
+
+        beforeEach(() => {
+            ORG_DATA["teams"] = MOCK_REPOS_TEAMS_DATA;
+        });
+
+        test('should set TEAMS_DATA object with no members or repositories', async () => {
+            const emptyValue = {};
+            const teamsData = { ... MOCK_TEAMS_DATA } as any;
+            teamsData[MOCK_REPOS_TEAMS_NAME]["members"] = emptyValue;
+            teamsData[MOCK_REPOS_TEAMS_NAME]["repositories"] = emptyValue;
+
+            spyFetchCall
+                .mockImplementationOnce(() => Promise.resolve({ json: () => Promise.resolve(emptyValue) } as any))
+                .mockImplementationOnce(() => Promise.resolve({ json: () => Promise.resolve(emptyValue) } as any));
+
+            await setTeamsData();
+
+            expect(TEAMS_DATA).toEqual(teamsData);
+
+            expect(spyFetchCall).toHaveBeenCalledTimes(2);
+            expect(spyFetchCall).toHaveBeenCalledWith(membersUrl, MOCK_HEADERS);
+            expect(spyFetchCall).toHaveBeenCalledWith(repositoriesUrl, MOCK_HEADERS);
+
+            expect(spyConsoleLog).toHaveBeenCalledTimes(1);
+            expect(spyConsoleLog).toHaveBeenCalledWith(`Get members info for ${MOCK_REPOS_TEAMS_NAME} team`);
```

Comment:
Minor refactor suggestion. Extract this `console.log` string into it's own variable at top of describe block scope to maintain consistency with other `expect` calls and de-clutter the `toHaveBeenCalledWith` call:

```
// Declare at top of describe block scope
const consoleLogMessage = `Get members info for ${MOCK_REPOS_TEAMS_NAME} team`;

...

// On lines 225 and 244 
expect(spyConsoleLog).toHaveBeenCalledWith(consoleLogMessage);